src/coin-gecko/api.ts

```diff
@@ -40,3 +41,26 @@ export function useGetRosePrice() {
     },
   )
 }
+
+export type TokenPriceInfo = {
+  price?: number
+  usedGecko?: boolean
```

Comment:
I would make it a required field :shrug:

Reply:
Done